src/content/docs/en/guides/astro-db.mdx

```diff
@@ -815,3 +819,31 @@ Additionally, [pushing any table schema changes](#pushing-table-schemas) (also k
 :::danger
 If you override your `.db` file on deployment, you will lose your production data. Follow the deployment method process for your host carefully to prevent data loss.
 :::
+
+## Migrate from Astro Studio to Turso
+
+<Steps>
+
+1. In the [Studio dashboard](https://studio.astro.build/), navigate to the project you wish to migrate. In the settings tab, use the "Export Database" button to download a dump of your database.
+2. Follow the official instructions to [install the Turso CLI](https://docs.turso.tech/cli/installation) and [sign up or log in](https://docs.turso.tech/cli/authentication) to your Turso account.
+3. Create a new database using the `.sql` dump you downloaded in step 1.
+   ```sh
+   turso db create [database-name] --from-dump ./path/to/dump.sql
+   ```
+4. Fetch the database URL using the Turso CLI, and use it as the environment variable `ASTRO_DB_REMOTE_URL`.
+   ```sh
+   turso db show [database-name]
+   ```
+   ```env
+   ASTRO_DB_REMOTE_URL=[your-database-url]
+   ```
+5. Create a token to access your database, and use it as the environment variable `ASTRO_DB_APP_TOKEN`.
+   ```sh
+   turso db tokens create [database-name]
+   ```
+   ```env
+   ASTRO_DB_APP_TOKEN=[your-app-token]
+   ```
```

Comment:
I wonder if the specific parts referencing the env vars should instead point to the libSQL guide instead? Something along the lines of "obtain the Turso database URL and token following [insert link to relevant Turso guide here] and then continue from the [link to libSQL guide] to connect Astro DB to Turso using libSQL.

Reply:
For the immediate case, I wouldn't want to send people to a bunch of different places or overwhelm them too much. Right now, I think a basic simple looking linear set of steps is immediately helpful, personally... People might already be freaking out and this very simple set of steps looks achievable without too much pain. 😄